### PR TITLE
Add release wording extraction explanations

### PR DESCRIPTION
**Pull request by lartist**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Add release wording extraction explanations
| Fixed ticket |

### CODE REVIEW

File: content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md [hunk 1]
```diff
@@ -73,6 +73,16 @@ Ask a maintainer from the PrestaShop Company with access to the Translation Tool
 It is usually only done once per release as well.
 {{% /notice %}}
 
+1. [Use the command to extract wordings from the TranslationTool repository](https://github.com/PrestaShopCorp/TranslationTool/actions/workflows/create-default-catalog-pr.yml). This command will automatically generates a Pull Request on the PrestaShop/PrestaShop repository (author should be jarvis). It is important to note that this PR must be reviewed by a member of the content team.
+
```
Comment (matks) [inline content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md:77]: Can you please add a line below that mentions that https://github.com/PrestaShopCorp/TranslationTool is PRIVATE so people need to be granted access to it to be able to use it 😉 ?

**Review by matks — COMMENTED** [review]
Good for me, only 1 thing to add

File: content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md [hunk 2]
```diff
@@ -73,6 +73,16 @@ Ask a maintainer from the PrestaShop Company with access to the Translation Tool
...
+2. If the team content member requests wording corrections, they can be found either in the Prestashop CORE, a module or in the directory /mails of the Prestashop CORE.
+
+3. If the correction occurs in a module, a release must be made, along with a bump in the composer.json of the CORE, before re-extracting.
+
+4. If the wording to be corrected is found in /mails, you should first correct it in https://github.com/PrestaShop/mjml-theme-converter and the mail templates in the Prestashop CORE should also be regenerated (using https://github.com/PrestaShop/mjml-theme-converter ).
+
+5. Once all the wordings have been corrected, validated, merged, module released, mail regenerated, bumping done, [a Github Action can be used on the TranslationTool repository](https://github.com/PrestaShopCorp/TranslationTool/actions/workflows/create-default-catalog-pr.yml) to push the catalogs to Crowdin (the reposiroty need right access).
```
[the remark this answers is not in the document]
Comment (jolelievre) [reply content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md:84]: I believe it's a different action for Crowdin push no?

File: content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md (hunk 1 above)
Comment (lartist) [reply content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md:78]: Done

File: content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md [hunk 3]
```diff
@@ -73,6 +73,16 @@ Ask a maintainer from the PrestaShop Company with access to the Translation Tool
 It is usually only done once per release as well.
 {{% /notice %}}
 
+1. [Use the command to extract wordings from the TranslationTool repository](https://github.com/PrestaShopCorp/TranslationTool/actions/workflows/create-default-catalog-pr.yml). This command will automatically generates a Pull Request on the PrestaShop/PrestaShop repository (author should be jarvis). It is important to note that this PR must be reviewed by a member of the content team.
+
+2. If the team content member requests wording corrections, they can be found either in the Prestashop CORE, a module or in the directory /mails of the Prestashop CORE.
+
+3. If the correction occurs in a module, a release must be made, along with a bump in the composer.json of the CORE, before re-extracting.
+
+4. If the wording to be corrected is found in /mails, you should first correct it in https://github.com/PrestaShop/mjml-theme-converter and the mail templates in the Prestashop CORE should also be regenerated (using https://github.com/PrestaShop/mjml-theme-converter ).
```
[the remark this answers is not in the document]
Comment (lartist) [reply content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md]: Done

**Review by matks — APPROVED** [review]
Thank you @lartist

**Pull request merged**